images/job-run-aggregator/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 LABEL maintainer="deads@redhat.com"
```

Comment:
Small issue here, you forgot to use `microdnf`

Reply:
Looks like added recently after rebase. Updated.